docs/eventing/samples/helloworld/helloworld-python/README.md

```diff
@@ -0,0 +1,338 @@
+    apiVersion: v1
+    kind: Namespace
+    metadata:
+      name: knative-samples
+      labels:
+           knative-eventing-injection: enabled
+    ---
+    # Helloworld-python app deploment
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: helloworld-python
+      namespace: knative-samples
+    spec:
+      replicas: 1
+      selector:
+        matchLabels: &labels
+          app: helloworld-python
+      template:
+        metadata:
+          labels: *labels
+        spec:
+          containers:
+            - name: helloworld-python
+              image: docker.io/{username}/helloworld-python
+              imagePullPolicy: IfNotPresent
+    ---
+    # Service that exposes helloworld-python app.
+    # This will be the subscriber for the Trigger
+      kind: Service
+      apiVersion: v1
+      metadata:
+        name: helloworld-python
+        namespace: knative-samples
+      spec:
+        selector:
+          app: helloworld-python
+        ports:
+        - protocol: TCP
+          port: 80
+          targetPort: 8080
+    ---
+    # Knative Eventing Trigger to trigger the helloworld-python service
+    apiVersion: eventing.knative.dev/v1alpha1
+    kind: Trigger
+    metadata:
+      name: helloworld-python
+      namespace: knative-samples
+    spec:
+      broker: default
+      filter:
+        attributes:
+          type: dev.knative.samples.helloworld
+          source: dev.knative.samples/helloworldsource
+      subscriber:
+        ref:
+          apiVersion: v1
+          kind: Service
+          name: helloworld-python
+    ```
+
+## Building and deploying the sample
+
+Once you have recreated the sample code files (or used the files in the sample
+folder) you're ready to build and deploy the sample app.
+
+1. Use Docker to build the sample code into a container. To build and push with
+   Docker Hub, run these commands replacing `{username}` with your Docker Hub
+   username:
+
+   ```shell
+   # Build the container on your local machine
+   docker build -t {username}/helloworld-python .
+
+   # Push the container to docker registry
+   docker push {username}/helloworld-python
+   ```
+
+1. After the build has completed and the container is pushed to docker hub, you
+   can deploy the sample application into your cluster. Ensure that the container image value
+   in `sample-app.yaml` matches the container you built in the previous step. Apply
+   the configuration using `kubectl`:
+
+   ```shell
+   kubectl apply --filename sample-app.yaml
+   ```
+    1.  Above command created a namespace `knative-samples` and labelled it with `knative-eventing-injection=enabled`, to enable eventing in the namespace. Verify using the following command:
+        ```shell
+        kubectl get ns knative-samples --show-labels
+        ```
+    1. It deployed the helloworld-python app as a K8s Deployment and created a K8s service names helloworld-python. Verify using the following command.
```

Comment:
What is it? Again use present tense "X deploys...as a...and creates a..."

"Verify that the previous step has been successful by entering the following command:"

I think the structure here overall for the steps could use some work. It's confusing that each step constantly refers back to a previous step. The verification should be part of the step it's verifying maybe?